plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1852,35 +1865,37 @@ int amdgpu_plugin_restore_file(int id)
 		return -ENOMEM;
 	}
 
-	if (read_file(img_path, buf, filestat.st_size)) {
+	if ((ret = read_file(img_path, buf, filestat.st_size))) {
```

Comment:
I believe the kernel's checkpatch would complain about this style. Personally I find this more reabable:

    ret = read_file(...);
    if (ret) {
        ...
    }

Reply:
ACK